Cargo.toml

```diff
@@ -12,3 +12,4 @@ nom = "6.1.0"
 num-complex = "0.3.0"
 petgraph = "0.5.1"
 serde = { version = "1.0.125", features = ["derive"] }
+thiserror = "1.0.26"
```

Comment:
Can you provide an MR description pitching this change?

Reply:
removed